docs/concepts/copilot/copilot-ai-capabilities.md

```diff
@@ -0,0 +1,169 @@
+4. You can keep or reject the suggested Pipeline and interactively execute it to see the results.
+5. View Data Copilot's suggested changes in the visual editor.
+
+### Modify an existing Pipeline
+
+You can also call Data Copilot to modify an existing Model. Select which Gem should be the starting point for modifications downstream, and type a new text prompt. Data Copilot will suggest a new sequence of data transformations after the selected starting point.
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/a2x6kkzy8y?videoFoam=true" title="Modify an existing Pipeline" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
+## Next-transformation suggestions
+
+Data Copilot can suggest the next transformation in a series or the next expression within a Gem.
+
+### Suggest Gems
+
+Data Copilot can suggest the next transformation in a series as long as there is an open Tail node.
+
+![Suggest Gems](img/copilot_next_suggestion.png)
+
+See the following Join suggestion example:
+
+1. Select a Dataset of interest.
+2. Data Copilot suggests Datasets which are frequently used with the selected Dataset.
+3. Data Copilot then suggests a next transformation, in this case, a Join Gem.
+
+### Suggest Expressions
+
+At the more granular level, Data Copilot can suggest expressions within Gems.
+
+![Suggest expressions](img/copilot_next_suggestion_expression.png)
+
+Within our [advanced Expression Builder](/Spark/expression-builder) you can:
+
+1. Type an English text prompt.
+2. Data Copilot generates a code expression for a particular column.
+3. Click to accept the code expression or try again with a different prompt.
+4. Run the Pipeline up to and including this Gem, and observe the resulting data sample.
+
+## Generate with AI
+
+Data Copilot can generate scripts, user-defined functions, or functions for macro within a Gem.
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/vhf63tjnun?videoFoam=true" title="Modify an existing Pipeline" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
```

Comment:
This doesn't seem like the correct function 
![image](https://github.com/SimpleDataLabsInc/prophecy-docs/assets/20563764/d293efcf-9c7d-413c-8a91-83ad4299f934)

I asked databricks copilot, and here's the correct answer: 
<img width="395" alt="image" src="https://github.com/SimpleDataLabsInc/prophecy-docs/assets/20563764/b1e99bc3-1ecd-4fdb-b618-84e3e9c75497">